code/game/atoms.dm

```diff
@@ -146,8 +146,10 @@
 
 	to_chat(user, "\icon[src] That's [f_name] [suffix]")
 	to_chat(user, desc)
-	if(description_fluff)
+	if(description_info)
 		to_chat(user, span("notice", "This item has additional examine info. <a href=?src=\ref[src];examine=fluff>\[View\]</a>"))
+	if(description_fluff)
+		to_chat(user, span("notice", "This item has additional lore info. <a href=?src=\ref[src];examine=fluff>\[View\]</a>"))
```

Comment:
I'm kinda wondering something.

Let's say there's an item with both info. They'll get two lines. But the link does exactly the same thing. It opens the exactly the same menu. There is no difference whether an item has extra "examine" info or extra "lore" info. Shit's the same.

Proposal, just do this:
```dm
	if(description_info || description_fluff)
		to_chat(user, span("notice", "This item has additional examine info. <a href=?src=\ref[src];examine=fluff>\[View\]</a>"))
```